docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -538,14 +539,19 @@ Requirements:
 - `to` either is an account contract or supports the <<IERC721Receiver,IERC721Receiver>> interface.
 
 [.contract-item]
-[[ERC721-_set_token_uri]]
-==== `[.contract-item-name]#++_set_token_uri++#++(ref self: ContractState, token_id: u256, token_uri: ByteArray)++` [.item-kind]#internal#
+[[ERC721-_set_base_uri]]
+==== `[.contract-item-name]#++_set_base_uri++#++(ref self: ContractState, base_uri: ByteArray)++` [.item-kind]#internal#
 
-Internal function that sets the `token_uri` of `token_id`.
+Internal function that sets the `base_uri`.
 
-Requirements:
+[.contract-item]
+[[ERC721-_base_uri]]
+==== `[.contract-item-name]#++_base_uri++#++(self: @ContractState) -> ByteArray++` [.item-kind]#internal#
 
-- `token_id` exists.
+Base URI for computing <<IERC721Metadata-token_uri, token_uri>>.
+
+If set, the resulting URI for each token will be the concatenation of the base URI and the token ID.
```

Comment:
should we add a small example?

Reply:
now that i think of it, this belongs to `token_uri`. this method is unaware of any concatenation, while the comment is relevant for `token_uri`